alacritty/src/event.rs

```diff
@@ -438,6 +439,22 @@ impl<N: Notify> Processor<N> {
         self.write_ref_test_results(&terminal.lock());
     }
 
+    fn paste<T: EventListener, A: crate::input::ActionContext<T>>(ctx: &mut A, contents: &str) {
```

Comment:
I have no idea where to place this function, we can make it free or keep here. What do you think?

Reply:
We could put this on the `Term` and pass a `<W: io::Write>`, similar to what we do with ansi handlers?

That would also allow potential consumers of `Term` to make use of Alacritty's support for bracketed pasting.

Reply:
The problem is, that we are getting terminal from ActionContext and io::Write should be implemented for it too, however we're ending up with &mut for Writer(rust requirements) and &ctx for terminal, so borrow checker just doesn't allow this code(even its 100% safe).

Generally this type of questions can be solved with RefCell etc, however this is not the case, unfortunately.

So the solution could be: create WriteTerm trait (e.g.) to reflect immutability, or to pass `write_to_pty` directly to term and use it for paste there.

***
Or just leave things as it is.

Reply:
> Or just leave things as it is.

I think that it makes sense where it is. Let's just go with this.

Just please don't put it in between the event loop and the event handler, I want to keep those right below each other.